### PR TITLE
feat: add all data to chart tooltip

### PR DESCRIPTION
**Pull request by Zerdados**
- Adds all data from an entry to the chart tooltip
- Also updates dependencies and fixes security vulnerability

Preview here: https://deploy-preview-148--open-data-dashboard.netlify.app/de/resource/hunde-insgesamt-5?visualization=barChart

### CODE REVIEW

File: src/components/visualization/bar-chart/BarChart.tsx [hunk 1]
```diff
@@ -81,6 +88,27 @@ export default function BarChart({
   );
 }
 
+const CustomTooltip: React.FC<CustomTooltipProps> = (props) => {
```
Comment (danielptv) [inline src/components/visualization/bar-chart/BarChart.tsx:91]: Is it possible to extract this one to a separate component?

**Review by danielptv — COMMENTED** [review]
Look and works great! Good job! 👍  Just one minor comment.

File: src/components/visualization/bar-chart/CustomTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { DefaultTooltipContent, type TooltipProps } from 'recharts';
+import { computeIfAbsent } from '@/utils/mapUtils';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+interface CustomTooltipProps extends TooltipProps<any, any> {
+  axesMap: Map<string, Map<string, boolean>>;
+  xAxis: string;
+}
+
+export const CustomTooltip: React.FC<CustomTooltipProps> = (props) => {
```
Comment (danielptv) [inline src/components/visualization/bar-chart/CustomTooltip.tsx:10]: Can this be defined as a "normal" React component (`export default function CustomTooltip ...`)?

File: src/components/visualization/bar-chart/CustomTooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { DefaultTooltipContent, type TooltipProps } from 'recharts';
+import { computeIfAbsent } from '@/utils/mapUtils';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+interface CustomTooltipProps extends TooltipProps<any, any> {
```
Comment (danielptv) [inline src/components/visualization/bar-chart/CustomTooltip.tsx:5]: If you change `any` to `unknown` you probably won't need the eslint-disable

**Review by danielptv — APPROVED** [review]
Looks good

File: src/components/visualization/bar-chart/CustomTooltip.tsx (hunk 3 above)
Comment (Zerdados) [reply src/components/visualization/bar-chart/CustomTooltip.tsx:5]: Unfortunately unknown doesn't fulfill the constraints that TooltipProps excepts to be fulfilled here. Avoiding any would probably require some quite complicated typing here

File: src/components/visualization/bar-chart/CustomTooltip.tsx (hunk 2 above)
Comment (Zerdados) [reply src/components/visualization/bar-chart/CustomTooltip.tsx:10]: Ah, sure

**Pull request merged**